src/basic/escape.c

```diff
@@ -119,7 +119,7 @@ char *cescape(const char *s) {
         return cescape_length(s, strlen(s));
 }
 
-int cunescape_one(const char *p, size_t length, char *ret, uint32_t *ret_unicode) {
+int cunescape_one(const char *p, size_t length, char *ret, char32_t *ret_unicode) {
```

Comment:
This code has been changed anyway in git, needs rebasing. The new code looks quite different.